### PR TITLE
Multipage

### PR DESCRIPTION
**Pull request by ferose**
Turn into a multi page app and add 404 and support page.

### CODE REVIEW

**Comment by shurane** [comment]
This is kinda cool. Did you want me to take a look?

File: src/pages/support/Support.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+interface ISupportProps {}
+interface ISupportState {}
+
+export class Support extends React.Component<ISupportProps, ISupportState> {
+    public render() {
+        return (
+            <div className="Support">
+                <h1>Support</h1>
+                <p>This is a beta site with a lot under constructions.</p>
+                <p>Feedbacks are appreciated with email: ferosegame@gmail.com</p>
```
Comment (shurane) [inline src/pages/support/Support.tsx]: Nit: "Feedback can be sent to". What do you think?

File: src/pages/support/Support.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+interface ISupportProps {}
+interface ISupportState {}
+
+export class Support extends React.Component<ISupportProps, ISupportState> {
+    public render() {
+        return (
+            <div className="Support">
+                <h1>Support</h1>
+                <p>This is a beta site with a lot under constructions.</p>
```
Comment (shurane) [inline src/pages/support/Support.tsx]: Nit: construction

File: src/App.tsx [hunk 3]
```diff
@@ -1,59 +1,58 @@
...
-            </Button>
-
-            <Menu
-                show={this.state.modalShow}
-                onHide={() => this.setState({modalShow: false})}
-            />
+                <Banner/>
+                <div className="main-content">
+                <Switch>
+                    <Route path="/" component={FourPlay} exact/>
```
Comment (shurane) [inline src/App.tsx]: Will this be ok as we keep adding games to the list? Like when it becomes as long as 10 games or 50 games?

File: src/pages/support/Support.tsx (hunk 1 above)
Comment (ferose) [reply src/pages/support/Support.tsx]: I like it

File: src/App.tsx (hunk 3 above)
Comment (ferose) [reply src/App.tsx]: Maybe, it's not going to load all the games at once. Just going to be a lot of games in that one file. We can probably figure out a solution to it when it becomes a problem.

Comment (shurane) [reply src/App.tsx]: I meant more like should we have an array of route objects and render those in a for loop?

**Pull request merged**